SWLOR.Game.Server/Item/Cooking/Drink.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Linq;
+using SWLOR.Game.Server.NWN;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.Item.Contracts;
+using SWLOR.Game.Server.NWN.Enum;
+using SWLOR.Game.Server.Service;
+
+using SWLOR.Game.Server.ValueObject;
+
+namespace SWLOR.Game.Server.Item.Cooking
+{
+    public class Drink : IActionItem
```

Comment:
As mentioned before, let's move the cooking stuff to a different PR to avoid merging in dead code.